tools/debian/copyright

```diff
@@ -0,0 +1,24 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cockpit
+Source: https://github.com/cockpit-project/cockpit
+
+Files: *
+Copyright: 2013-2015  Red Hat, Inc.
```

Comment:
> Lets just say Cockpit Contributors

see @mbiebl 's [answer to this](https://github.com/cockpit-project/cockpit/pull/3202#issuecomment-161083889)